### PR TITLE
Added a Vue.js section for props

### PR DESCRIPTION
**Pull request by CarmeloNaciti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -664,8 +664,17 @@ export class SettingsComponent extends Component {
...
+```js
+Vue.component('child', {
+    props: ['class', 'loading', 'message'],
+    template: '<span v-bind:class="class" v-show="loading" v-text="message"></span>'
+})
+```
+In the template:
+```html
+<child class="small" loading="true" message="Busy Loading!"></child>
+```
```
Comment (feimosi) [inline README.md:893]: Could you make one prop dynamic e.g.  `:loading` ?

File: README.md [hunk 2]
```diff
@@ -664,8 +664,17 @@ export class SettingsComponent extends Component {
 ```
 
 ### Vue.js
-
-> TODO
+In the parent component:
+```js
+Vue.component('child', {
+    props: ['class', 'loading', 'message'],
```
Comment (feimosi) [inline README.md:886]: Please use props with validation.

**Review by feimosi — CHANGES_REQUESTED** [review]

**Comment by feimosi** [comment]
Would you mind adding Conditional rendering Vue example as well?

File: README.md (hunk 2 above)
Comment (CarmeloNaciti) [reply README.md:886]: Will have this sorted in the week.

File: README.md (hunk 1 above)
Comment (CarmeloNaciti) [reply README.md:893]: Will have this sorted in the week.

**Comment by feimosi** [comment]
@CarmeloNaciti any update?

**Comment by feimosi** [comment]
@CarmeloNaciti please update the PR.

**Comment by feimosi** [comment]
@CarmeloNaciti ?

**Pull request merged**